### PR TITLE
Feature/Point: `Neg`

### PR DESCRIPTION
**Pull request by SherLzp**
- Add `Neg` function for std/twistededwards/point
- Add tests for `Neg` function

### CODE REVIEW

File: std/algebra/twistededwards/point.go [hunk 1]
```diff
@@ -171,3 +175,33 @@ func (p *Point) ScalarMulFixedBase(cs *frontend.ConstraintSystem, x, y interface
...
+	case ecc.BLS12_381:
+		r = cs.Constant(bls12381fr.Modulus())
+		break
+	case ecc.BLS12_377:
+		r = cs.Constant(bls12377fr.Modulus())
+		break
+	case ecc.BW6_761:
+		r = cs.Constant(bw6761fr.Modulus())
+		break
+	default:
```
Comment (gbotrel) [inline std/algebra/twistededwards/point.go:197]: need to panic on default case when curve is not supported / implemented yet
For example we just added BLS24-315 curve --

Comment (SherLzp) [reply std/algebra/twistededwards/point.go:200]: that's right. Already add panic.

**Comment by gbotrel** [comment]
@SherLzp thanks for the submission. Sorry for the delay to merge that, we have few other tasks right now and need to discuss if the `Neg` operation shouldn't be in the `cs` API like the `Inv` one.

**Comment by SherLzp** [comment]
Noted

**Pull request merged**